schemas/logs/aws/alb.yml

```diff
@@ -23,13 +24,13 @@ fields:
     type: string
   - name: clientPort
     description: The port of the requesting client.
-    type: int
+    type: bigint
```

Comment:
It seems that these types are changing again. I know that in theory it shouldn't matter, but is that expected to happen again @alxarch

Reply:
Yes this is the `push` part of the mage PR. This is expected.